.circleci/config.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+ jobs:
+   build:
+     docker:
+       - image: circleci/ruby:2.4.1
```

Comment:
Why use a ruby image here? You'll probably want to use node

Reply:
Good call! I would add my node version into the image, correct?